Bottom: 45be9a328a8137afd11b25069407cf28a152c4bd
Top:    610b7f0cfe3fc0dfb6ef5829a4f1b2fe30b7c722
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-15 17:13:21 -0500

Refresh of add-buildbot-role

---
diff --git a/roles/buildbot-master/defaults/main.yml b/roles/buildbot-master/defaults/main.yml
index 43907def..8806961b 100644
--- a/roles/buildbot-master/defaults/main.yml
+++ b/roles/buildbot-master/defaults/main.yml
@@ -64,6 +64,8 @@ container_name: '{{ docker_name_prefix }}buildbot'
 # networks of the container
 container_networks:
   - name: '{{ docker_network_frontend }}'
+    aliases:
+      - '{{ buildbot_hostname }}'
 
 # volumes mounted within the container
 container_volumes:
